src/index.ts

```diff
@@ -39,19 +43,47 @@ githubApp.webhooks.on("workflow_job.queued", async (event) => {
                 status: "queued"
             })
         };
-    response = await fetch(database+`/api/config_repos/${event.payload.repository.name}`, requestOptions);
-    
+
+    var response = await fetch(database+`/api/config_repos/${event.payload.repository.name}`, requestOptions);
+
     // call API to compute service
+    // generate token based on org name and repository
+    const secretKey = await fs.readFile(path.join(__dirname, 'src/middleware/custom-token.pem')) 
```

Comment:
Could you please add instructions to generate the key into the README?

Reply:
Added readme.md